library/src/scala/runtime/Scala3RunTime.scala

```diff
@@ -10,4 +10,10 @@ object Scala3RunTime:
   def assertFailed(): Nothing =
     throw new java.lang.AssertionError("assertion failed")
 
+  // Called by the inline extension def nn. Extract to minimize the bytecode size at call site.
```

Comment:
It looks like this should be the documentation comment

Reply:
Done.